services/opt/service.go

```diff
@@ -0,0 +1,58 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package opt
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/containerd/containerd/plugin"
+	"github.com/pkg/errors"
+)
+
+// Config for the opt manager
+type Config struct {
+	// Path for the opt directory
+	Path string `toml:"path"`
+}
+
+func init() {
+	plugin.Register(&plugin.Registration{
+		Type: plugin.InternalPlugin,
+		ID:   "opt",
+		Config: &Config{
+			Path: "/opt/containerd",
```

Comment:
Should we add OS specific defaults i.e. Windows, etc?  Although I'm not sure what the default install path is for Windows.

Reply:
@johnstep can you help us out on how to make this work on windows?  The idea is that we modify the daemon's %PATH% to some type of managed directory that we can install binaries to

Reply:
Sure. Is there an expectation that only one containerd will be installed on a given Windows system? In other words, should it be a global directory, or under the containerd root directory? I lean toward the latter.